mindmap/mindmap.py

```diff
@@ -22,11 +24,31 @@ class MindMapXBlock(XBlock):
         help="A simple counter, to show something happening",
     )
 
+    display_name = String(
+        display_name="Display Name",
+        default="Mind Map",
+        scope=Scope.settings,
+    )
```

Comment:
For now, I understand this is the only configuration possible for the Xblock, but are there any other can we consider for the near future?

Reply:
Yes, the library has some options that allow you to customize some basic aspects. Some features that could be included are:
- Allow students to select a custom theme.
- Allow students to customize the mind map, such as: Setting curved or straight lines, or the color of the lines.
- Allow the instructor to lock the editing of the Mind Map.
- Allow the instructor/student to customize the keyboard shortcuts to use the tool.

You can see all the options in [this link](https://github.com/hizzgdev/jsmind/blob/master/docs/en/2.options.md)